tools/cpp/bsd_cc_toolchain_config.bzl

```diff
@@ -57,20 +57,20 @@ all_link_actions = [
 def _impl(ctx):
     cpu = ctx.attr.cpu
     compiler = "compiler"
-    toolchain_identifier = "local_freebsd" if cpu == "freebsd" else "stub_armeabi-v7a"
-    host_system_name = "local" if cpu == "freebsd" else "armeabi-v7a"
-    target_system_name = "local" if cpu == "freebsd" else "armeabi-v7a"
-    target_libc = "local" if cpu == "freebsd" else "armeabi-v7a"
-    abi_version = "local" if cpu == "freebsd" else "armeabi-v7a"
-    abi_libc_version = "local" if cpu == "freebsd" else "armeabi-v7a"
+    toolchain_identifier = "local_{}".format(cpu) if cpu == "freebsd" or cpu == "openbsd" else "stub_armeabi-v7a"
```

Comment:
I'd do:

`is_bsd = cpu == "freebsd" || cpu == "openbsd"`

and then use `is_bsd` in all lines to avoid duplication and ensure they all remain consistent. Would also help cut this long line.

Reply:
Much better. Done.